tests/contrib/storage/test_linux_ceph.py

```diff
@@ -1602,3 +1618,13 @@ def test_add_op_create_pool(self):
         op = base_op.copy()
         op['app-name'] = 'earth'
         self.assertEqual(rq.ops, [op])
+        rq = ceph_utils.CephBrokerRq()
+        rq.add_op_create_pool('apool', max_bytes=42)
+        op = base_op.copy()
+        op['max-bytes'] = 42
+        self.assertEqual(rq.ops, [op])
+        rq = ceph_utils.CephBrokerRq()
+        rq.add_op_create_pool('apool', max_objects=42)
```

Comment:
It would have been better to use a different value to ensure no mismatch

Reply:
Thanks for the input @sahid. I certainly agree for when using mock.assert_called_with statements multiple times so you know which iteration of the call you are validating.  But since I am recreating the datastructure each time and using assertEqual I do believe this should be safe.